crates/interpreter/src/interpreter/contract.rs

```diff
@@ -60,6 +65,7 @@ impl Contract {
             bytecode,
             hash,
             contract_address,
+            None,
```

Comment:
This should be added from `env.tx.kind` if kind is `To` we should set that address.

Reply:
I think I get your meaning. I set it to None in the case of create and to the destination address in the case of Call